### ISSUE TITLE
This week should return range from today till end of this week instead of single date.

### ISSUE THREAD

**Issue by Kanagaraj17** [facebook/duckling#456]
Input : this week
Output from Ducking: Monday, 24 February 2020(which is already over)

Expected Output: It should be range from Tomorrow to the end of this week(say if today is Wednesday, It should be from today to the end of this week, Saturday.)

Thanks in Advance.

**Comment by chessai** [comment]
I agree that this should probably be a Duration. In natural speech I wouldn't say "this week" to refer to the start of the week, but rather the duration of the week.
